gitlab/sync.py

```diff
@@ -0,0 +1,189 @@
+from agilitysync.sync import (
+    as_exceptions,
+    BaseInbound,
+    BaseOutbound,
+    BasePayload,
+    BaseEvent,
+    BaseAttachmentUpload,
+    FieldTypes,
+    EventTypes,
+    EventCategory
+)
+import re
+from datetime import datetime
+from dateutil import parser
+import time
+import json
+from external_plugins.gitlab import transformer_functions
+
+
+class Payload(BasePayload):
+
+    def fetch_project(self, event):
+        project = str(event["project"]["id"])
+        org = event['project']['namespace']
+        return org + "/" + project
+    
+    def fetch_asset(self, event):
+        return "Assettype-001"
```

Comment:
Map asset and return value

Reply:
api dosnt provide value for the asset,so i gace the default name asserttype-001 for issue in gilab